source/compiler-core/slang-spirv-core-grammar.h

```diff
@@ -4,7 +4,13 @@
 #include "../core/slang-string.h"
 #include "../core/slang-string-slice-pool.h"
 #include "../core/slang-dictionary.h"
+
+#ifdef SLANG_EXTERNAL_SPIRV_HEADER
+#include <spirv/unified1/spirv.h>
```

Comment:
Can you make it so it's just unconditionally `<spirv/unified1/spirv.h>` please, it might just work, but if not you'll need to add the appropriate target to the private dependencies of whatever target builds this.

Reply:
and same for the repeat of this pattern.